requirements/base.txt

```diff
@@ -46,13 +46,13 @@ jinja2==3.1.2
     # via code-annotations
 markupsafe==2.1.1
     # via jinja2
-newrelic==8.2.0.181
+newrelic==8.2.1
     # via edx-django-utils
-openedx-events==1.0.0
+openedx-events==2.0.0
```

Comment:
A [breaking change](https://github.com/openedx/openedx-events/releases/tag/v2.0.0) in the new version, needs to be looked further before merging.

Reply:
This should be safe to update everywhere -- it's a removal of something that should already have been deleted everywhere anyhow.